autogen.pl

```diff
@@ -979,6 +979,28 @@ sub patch_autotools_output {
 
         # We have to change the search pattern and substitution on each
         # iteration to take into account the tag changing
+        my $search_string = '# icc used to be incompatible with GCC.\n\s+' .
```

Comment:
did we really have to introduce one more foreach loop? Couldn't the one that looks for Sun's compiler be reused?

Reply:
this is the way we have always done. it makes it easier to follow the logic of what we are doing.

an empty line was removed by mistake, and i fixed that